### PR TITLE
New link on invoice page

### PR DESCRIPTION
**Pull request by grana-equinix**
[#369](https://github.com/killbill/killbill-admin-ui/issues/369)

Adds a new link on invoice page via customization approach to redirect to a new screen. the new screen is not available on open-source. 

Customize app with a new link as below
Screenshot 2023-12-08 at 4 55 01 PM

Open-source no visible change as below 
Screenshot 2023-12-08 at 5 03 16 PM

### CODE REVIEW

File: app/views/kaui/invoices/show.html.erb [hunk 1]
```diff
@@ -7,6 +7,9 @@
       <% unless @invoice.invoice_number.blank? %>
         <%= Kaui.customer_invoice_link.call(@invoice, self) %>
       <% end %>
+      <% unless @invoice.invoice_number.blank? %>
```
Comment (pierre) [inline app/views/kaui/invoices/show.html.erb:10]: You can simplify and remove this (one `unless` for both calls).

**Review by pierre — CHANGES_REQUESTED** [review]
Don't forget also to update the docs in `killbill-admin-ui-standalone` repo (including an example of validate HTML for that new customization).

**Comment by grana-equinix** [comment]
> Don't forget also to update the docs in `killbill-admin-ui-standalone` repo (including an example of validate HTML for that new customization).

Done killbill-admin-ui-standalone change [link](https://github.com/killbill/killbill-admin-ui-standalone/pull/65)

File: app/views/kaui/invoices/show.html.erb [hunk 2]
```diff
@@ -7,6 +7,9 @@
...
+        <%= Kaui.customer_bulk_invoice_adjust_link.call(@invoice, self) %>
```
[the remark this answers is not in the document]
Comment (grana-equinix) [reply app/views/kaui/invoices/show.html.erb]: Done

File: app/views/kaui/invoices/show.html.erb (hunk 1 above)
Comment (grana-equinix) [reply app/views/kaui/invoices/show.html.erb]: Combined the both logic in same condition.

**Review by pierre — APPROVED** [review]

**Pull request merged**